tests/scripts/hpo/hpo_multiple_exp_tests.sh

```diff
@@ -240,7 +240,12 @@ function hpo_multiple_exp_test() {
 
 			http_code=$(tail -n1 <<< "${get_trial_json}")
 			response=$(echo -e "${get_trial_json}" | tail -2 | head -1)
-			response=$(echo ${response} | cut -c 4-)
+
+			# Added condition to check for '000' as sometimes cURL command returns it due to reasons such as
+			# 'Failed DNS resolution','connection refused' or 'timed out'
+			if [ ${response::3} == "000" ]; then
+				response=$(echo ${response} | cut -c 4-)
+			fi
```

Comment:
Looks like this is common code that is used in multiple places. Can you create a function for this instead?